rdmd.d

```diff
@@ -495,10 +507,20 @@ private int rebuild(string root, string fullExe,
             ~ [ "-of" ~ fullExeTemp ]
             ~ [ "-od" ~ objDir ]
             ~ [ "-I" ~ dirName(root) ]
+            ~ [ "-i" ]
+            ~ [ "-v" ]
```

Comment:
I assume this was just for development / testing?

Reply:
Actually it's necessary.  Before this change the compiler was called using `-v`, now both calls are combined into a single call.  rdmd still uses the output from `-v` to read the dependencies to know when a rebuild is necessary.

Reply:
Ah fair enough. Thanks for the explanation!